db/seeds.rb

```diff
@@ -16,3 +16,199 @@
+  },
+  {
+    text: "Nature often recovers from even severe perturbations",
+  },
+  {
+    text: "Conservation goals should be based on science",
+  },
+  {
+    text: "Protecting nature for its own sake does not work",
+  },
+  {
+    text: "There is no significant conservation value in highly modified landscapes",
+  },
+  {
+    text: "Conservation will only be a durable success if it has broad public support",
+  },
+  {
+    text: "Conservation should work with, not against, capitalism",
+  },
+  {
+    text: "Working with corporations is not just pragmatic; they can be a positive force for conservation",
+  },
+  {
+    text: "To achieve conservation goals, human population growth must be reduced",
+  },
+  {
+    text: "Human affection for nature grows in line with income",
+  },
+  {
+    text: "Advancing the wellbeing of all people should be a goal of conservation",
+  },
+  {
+    text: "Conservation should seek to reduce the emotional separation of people from nature",
+  },
+  {
+    text: "Conservation goals should be based on ethical values",
+  },
+  {
+    text: "Maintaining ecosystem processes should be a goal of conservation",
+  },
+  {
+    text: "Economic arguments for conservation are risky because they can lead to unintended negative conservation outcomes",
+  },
+  {
+    text: "Having multiple rationales for conservation weakens the conservation movement",
+  },
+  {
+    text: "Conservation messages that emphasise the value of nature for nature's own sake are more effective than those that promote the benefits of nature to humans",
+  },
+  {
+    text: "There is a risk that economic rationales for conservation will displace other motivations for conservation",
+  },
+  {
+    text: "Conservation communications are more effective when they use negative 'doom and gloom' messages rather than positive messages",
+  },
+  {
+    text: "Giving a voice to those affected by conservation actions improves conservation outcomes",
+  },
+  {
+    text: "To achieve its goals, conservation should seek to reform global trade",
+  },
+  {
+    text: "Non-native species offer little conservation value",
+  },
+  {
+    text: "Human impact on nature grows in line with incomes",
+  },
+  {
+    text: "Maintaining biological diversity should be a goal of conservation",
+  },
+  {
+    text: "Conservation will only be a durable success if it has the support of corporations",
+  },
+  {
+    text: "Conservation should seek to do no harm to poor people",
+  },
+  {
+    text: "Giving a voice to those affected by conservation action is an ethical imperative",
+  },
+  {
+    text: "The best way for conservation to contribute to human wellbeing is by promoting economic growth",
+  },
+  {
+    text: "When communities manage their own resources, their efforts are more effective than top-down approaches",
+  }
+]
+
+questions.each do |question|
+  Question.where(text: question[:text]).first_or_create do |q|
+    q.text = question[:text]
```

Comment:
Since we only have text, this line is unnecessary, we can probably simplify this statement a bit

Reply:
Thanks @spencerldixon I will do that!